src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -360,13 +367,36 @@ const Highchart = (() => {
             const low_barrier  = contract.low_barrier;
             if (barrier) {
                 addPlotLine({ id: 'barrier',      value: barrier * 1,      label: localize('Barrier ([_1])', [addComma(barrier)]),           dashStyle: 'Dot' }, 'y');
+                prev_barriers[0] = barrier; // Store values for future references.
             } else if (high_barrier && low_barrier) {
                 addPlotLine({ id: 'high_barrier', value: high_barrier * 1, label: localize('High Barrier ([_1])', [addComma(high_barrier)]), dashStyle: 'Dot' }, 'y');
                 addPlotLine({ id: 'low_barrier',  value: low_barrier * 1,  label: localize('Low Barrier ([_1])', [addComma(low_barrier)]),   dashStyle: 'Dot' }, 'y');
+                prev_barriers[0] = high_barrier; // Store values for future references.
+                prev_barriers[1] = low_barrier; // Store values for future references.
```

Comment:
can we change 0 to low and 1 to high? it's easier to remember since 1 is higher than 0 :P

Reply:
That was an impulsive decision 😅 Will change it.